homeassistant/components/datetime/__init__.py

```diff
@@ -74,21 +79,26 @@ class DateTimeEntityDescription(EntityDescription, frozen_or_thawed=True):
     """A class that describes date/time entities."""
 
 
-class DateTimeEntity(Entity):
+CACHED_PROPERTIES_WITH_ATTR_ = {
+    "native_value",
```

Comment:
`device_class` and `state_attributes` are omitted here because they don't need a setter